### ISSUE TITLE
How to generate the class-agnostic segmentation results (0-1 mask) based on the pre-trained model?

### ISSUE THREAD

**Issue by cjfcsjt** [mmaaz60/mvits_for_class_agnostic_od#23]
Thanks for your excellent work. Could you please tell me how to generate the masks of the saliency object given the pre-trained model?

**Comment by mmaaz60** [comment]
Hi @cjfcsjt,

Thank You for your interest in our work. MAVL only provides the class-agnostic bounding boxes. For salient object detection, we use the pretrained COCO Mask RCNN head to generate the segmentation masks given the salient object bounding boxes. Please look at the Section D.5 of [our paper](https://arxiv.org/pdf/2111.11430.pdf) for more details.

Thanks
